force_bdss/core/input_slot_info.py

```diff
@@ -22,6 +23,12 @@ class InputSlotInfo(HasStrictTraits):
     #: The user defined name of the variable containing the value.
     name = Identifier()
 
+    #: The CUBA key of the slot
+    type = CUBAType()
```

Comment:
We include `type` and `description` traits in both `InputSlotInfo` and `OutputSlotInfo` so that they contain all information from their corresponding `Slot` classes that is required by the WfManager UI. By doing so, we only need to refer to one object further down the line, rather than two.